samples/managed-app-sample/mainTemplate.json

```diff
@@ -0,0 +1,282 @@
+            "defaultValue": "azurevote",
+            "metadata": {
+                "description": "The name of the extension."
+            }
+        },
+        "clusterResourceName": {
+            "type": "String",
+            "metadata": {
+                "description": "The name of the Managed Cluster resource."
+            }
+        },
+        "location": {
+            "type": "String",
+            "metadata": {
+                "description": "The location of AKS resource."
+            }
+        },
+        "app-title": {
+            "type": "String"
+        },
+        "app-value1": {
+            "type": "String"
+        },
+        "app-value2": {
+            "type": "String"
+        },
+        "vmSize": {
+            "type": "String",
+            "defaultValue": "Standard_DS2_v2",
+            "metadata": {
+                "description": "VM size"
+            }
+        },
+        "vmEnableAutoScale": {
+            "type": "Bool",
+            "defaultValue": true,
+            "metadata": {
+                "description": "enable auto scaling"
+            }
+        },
+        "vmCount": {
+            "type": "Int",
+            "defaultValue": 3,
+            "metadata": {
+                "description": "VM count"
+            }
+        },
+        "dnsPrefix": {
+            "defaultValue": "[concat(parameters('clusterResourceName'),'-dns')]",
+            "type": "String",
+            "metadata": {
+                "description": "Optional DNS prefix to use with hosted Kubernetes API server FQDN."
+            }
+        },
+        "osDiskSizeGB": {
+            "defaultValue": 0,
+            "minValue": 0,
+            "maxValue": 1023,
+            "type": "Int",
+            "metadata": {
+                "description": "Disk size (in GiB) to provision for each of the agent pool nodes. This value ranges from 0 to 1023. Specifying 0 will apply the default disk size for that agentVMSize."
+            }
+        },
+        "kubernetesVersion": {
+            "type": "String",
+            "defaultValue": "1.24.0",
+            "metadata": {
+                "description": "The version of Kubernetes."
+            }
+        },
+        "networkPlugin": {
+            "defaultValue": "kubenet",
+            "allowedValues": [
+                "azure",
+                "kubenet"
+            ],
+            "type": "String",
+            "metadata": {
+                "description": "Network plugin used for building Kubernetes network."
+            }
+        },
+        "enableRBAC": {
+            "defaultValue": true,
+            "type": "Bool",
+            "metadata": {
+                "description": "Boolean flag to turn on and off of RBAC."
+            }
+        },
+        "enablePrivateCluster": {
+            "defaultValue": false,
+            "type": "Bool",
+            "metadata": {
+                "description": "Enable private network access to the Kubernetes cluster."
```

Comment:
Nit: Boolean flag to turn on and off private network access to the Kubernetes cluster. ?